packages/home-service/src/typedef.graphql

```diff
@@ -1,3 +1,24 @@
+scalar DateTime
+
+enum APIROLE {
+  ADMIN
+  USER
+}
+type UserType {
+  _id: String
+  name: String
+  title: String
+  uid: String
+  rhatUUID: String
+  memberOf: [String]
+  isActive: Boolean
+  apiRole: APIROLE
+  createdBy: String
+  createdOn: DateTime
+  updatedBy: String
+  updatedOn: DateTime
+}
+
```

Comment:
You don't need to define this duplicate

Read this https://www.apollographql.com/docs/apollo-server/federation/introduction/#separation-of-concerns

Reply:
Thanks